ui/apps/ui/src/app/pages/search-page/search-page.component.html

```diff
@@ -96,7 +96,7 @@
 
 <!--          </div>-->
 
-          <a href="{{marketplaceUrl}}user_documentation" class="right-media-link">
```

Comment:
wrong link, it should be:
 https://docs.sandbox.eosc-beyond.eu/
 ticket; https://github.com/cyfronet-fid/eosc-search-service/issues/1708